test/mocks/deps/one.js

```diff
@@ -14,3 +14,16 @@ let a = require('@a/package')
 let b = require('@b/package/method')
 let c = require('c/specific/method')
 let d = require('d')
+
+// Minifiers may redefine variables to save space
+let missingInit
+missingInit = 'ignoredmodule'
+require(missingInit)
+
+// When minifiers redefine variables, the original might be an invalid module string
+let requiredThing = 'some thing '
+requiredThing = 'validmodule'
+require(requiredThing)
```

Comment:
It seems like you want to support this but I'm not seeing it appear in the tests, what was the intent here? Sorry I'm not following!

Also can we change that from `'some thing '` to something else? We already have a match in there for `something` and I just want to be a bit clearer. Thanks!

Reply:
It's another case of minifiers reusing variables. Yes I can change to "invalid module string "